builtin/game/item.lua

```diff
@@ -205,6 +207,16 @@ function core.get_node_drops(nodename, toolname)
 	return got_items
 end
 
+local non_player_name = "!nonplayer"
```

Comment:
`:get_player_name()` returns an empty string for non-player ObjectRef, so that would be another possibility. Pipeworks already uses `:pipeworks` as dummy player but there's nowhere defined what dummy players are allowed to do. Thus, I'm not sure what the best solution here might be.